packages/list/components/List.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import styled from "react-emotion";
+import { listReset } from "../../shared/styles/styleUtils";
+import { listMarker } from "../style";
+
+export interface SharedListProps {
+  children:
+    | Array<React.ReactElement<HTMLLIElement>>
+    | React.ReactElement<HTMLLIElement>;
+  tag: "ul" | "ol";
+}
+
+interface ListProps extends SharedListProps {
+  /**
+   * The style of the text bullets. See https://www.w3.org/TR/CSS21/generate.html#list-style for more info
+   */
+  markerStyle?: React.CSSProperties["listStyleType"];
```

Comment:
Shouldn't this be defined by ui-kit for consistency?

Reply:
Yea, we could limit which values we accept in ui-kit